src/CourseDashboard.hpp

```diff
@@ -1,7 +0,0 @@
-#pragma once
-
-class CourseDashboard
```

Comment:
Why this is class is removed? It was intended to be a main class of the whole application.